glide-core/src/client/value_conversion.rs

```diff
@@ -44,6 +48,31 @@ pub(crate) fn convert_to_expected_type(
     };
 
     match expected {
+        ExpectedReturnType::ClusterScanReturnType {
+            cursor,
+            keys,
+        } => match value {
+            Value::Nil => Ok(value),
+            Value::Array(array) => {
+                if array.len() != 2 {
+                    return Err((
+                        ErrorKind::TypeError,
+                        "Array must contain exactly two elements",
+                    )
+                        .into());
+                }
+
+                let cursor = convert_to_expected_type(array[0].clone(), Some(*cursor))?;
+                let keys = convert_to_expected_type(array[1].clone(), Some(*keys))?;
```

Comment:
Second argument is `Option<ExpectedReturnType>` - for example `Some(ExpectedReturnType::String)`

Reply:
In the other example its as i did, why here not?